qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/WarmupIT.java

```diff
@@ -72,6 +73,12 @@ public void testKNNWarmupDefaultMethodFieldMapping() throws Exception {
     // Custom Method Field Mapping
     // space_type : "innerproduct", engine : "faiss", m : 50, ef_construction : 1024
     public void testKNNWarmupCustomMethodFieldMapping() throws Exception {
+
+        // Skip test if version is 1.0 or 1.1
+        Optional<String> bwcVersion = getBWCVersion();
+        if (bwcVersion.isEmpty() || bwcVersion.get().startsWith("1.0") || bwcVersion.get().startsWith("1.1")) {
```

Comment:
Why not move this to the gradle to get rid of if/else statement?

Reply:
updated